nuxt.config.ts

```diff
@@ -59,40 +91,13 @@ export default defineNuxtConfig({
   // https://v3.nuxtjs.org/api/configuration/nuxt.config#ignore
   i18n: {
     defaultLocale: 'de',
-    locales: [
-      {
-        code: 'de',
-        name: 'Deutsch',
-      },
-      {
-        code: 'en',
-        name: 'English',
-      },
-      {
-        code: 'es',
-        name: 'Español',
-      },
-      {
-        code: 'fr',
-        name: 'Français',
-      },
-      {
-        code: 'ja',
-        name: '日本語',
-      },
-    ],
+    locales: supportedLocales,
     vueI18n: {
       fallbackLocale: 'en',
       legacy: false,
       locale: 'de',
```

Comment:
We should standardise which locale we want to use as the default/fallback. Maybe we switch to 'en' for everything

Reply:
Yeah, I think we should as well. IIRC it was just `de` because that's what Buffet used when setting it up, just to make sure it was working. Now that we know it does, it should be good to flip back to `en` as the default.